src/lib/crypto/krb/prng_os.c

```diff
@@ -26,68 +26,49 @@
  */
 
 /*
- * This file implements a PRNG module which relies on the system's /dev/urandom
+ * This file implements a PRNG module which relies on the system's kernel
+ * PRNG, acccessed through getrandom() with fallback to the /dev/urandom
```

Comment:
This should be vaguer, since k5_os_getrandom() doesn't necessarily do what this describes (e.g. on Windows).  The comment at the top should also stay the same, I think.